feat: document wrapper methods (#81)
spicetify/spicetify-docs: docs/development/api-wrapper/classes/playbar.md

---
title: Playbar
description: Create buttons in the player.
---

Create buttons in the player, next to the player controls.

> This is useful for creating buttons which actions has an impact on/relating to the player and is generally dynamic/stateful, such as a button that toggles the player's loop mode.

Reviewer: ```suggestion
This is useful for creating buttons whose actions have an impact on or relate to the player, and are generally dynamic/stateful, such as a button that toggles the player's loop mode.
```
